src/winstanley/WdlAnnotator.scala

```diff
@@ -36,6 +36,14 @@ class WdlAnnotator extends Annotator {
         }
       } else
         ()
+    case declaration: WdlDeclaration =>
+      if (declaration.getSetter == null)
```

Comment:
I think once you merge your other PR everything (even inputs) will look be a declaration. So I think you'll want to add another "and we're not in an input block" requirement before the error highlight